at_lookup/lib/src/connection/outbound_message_listener.dart

```diff
@@ -48,23 +48,23 @@ class OutboundMessageListener {
       result = utf8.decode(_buffer.getData());
       result = result.trim();
       _buffer.clear();
-      _queue.addFirst(result);
+      _queue.add(result);
     }
   }
 
   /// Reads the response sent by remote socket from the queue.
-  /// If there is no message in queue after [maxWaitMilliSeconds], return null. Defaults to 30 seconds.
-  Future<String> read({int maxWaitMilliSeconds = 30000}) async {
+  /// If there is no message in queue after [maxWaitMilliSeconds], return null. Defaults to 3 seconds.
+  Future<String> read({int maxWaitMilliSeconds = 3000}) async {
     return _read(maxWaitMillis: maxWaitMilliSeconds);
   }
 
-  Future<String> _read({int maxWaitMillis = 30000, int retryCount = 1}) async {
+  Future<String> _read({int maxWaitMillis = 3000, int retryCount = 1}) async {
     String result;
     var maxIterations = maxWaitMillis / 10;
     if (retryCount == maxIterations) {
       _buffer.clear();
-      throw AtTimeoutException(
-          'No response after $maxWaitMillis millis from remote secondary');
+//      throw AtTimeoutException(
+//          'No response after $maxWaitMillis millis from remote secondary');
```

Comment:
Why has the throwing of AtTimeoutException been commented out?

Reply:
oops!! my bad. accidental checkin.